hostinit/setup.go

```diff
@@ -125,15 +139,25 @@ func (init *HostInit) startHosts(ctx context.Context) error {
 		})
 	}
 
-	grip.Info(message.Fields{
-		"GUID":      init.GUID,
-		"runner":    RunnerName,
-		"method":    "startHosts",
-		"num_hosts": started,
-		"runtime":   time.Since(startTime),
-	})
+	m := message.Fields{
+		"GUID":       init.GUID,
+		"runner":     RunnerName,
+		"method":     "startHosts",
+		"num_hosts":  started,
+		"total":      len(hostsToStart),
+		"runtime":    time.Since(startTime),
+		"had_errors": false,
+	}
 
-	return nil
+	if catcher.HasErrors() {
+		m["errors"] = catcher.Resolve()
+		m["had_errors"] = true
+	}
+
+	grip.CriticalWhen(catcher.HasErrors(), m)
```

Comment:
Again, nit, but would it better to roll these two in to the if statement above?

Reply:
The reason for two statements are: 

- don't add an errors field if its going to be nil, 
- log at different levels based on error contents.

Reply:
This is what I was thinking of:
```go
if catcher.HasErrors() {
	m["errors"] = catcher.Resolve()
	m["had_errors"] = true
	grip.Critical(m)

}else {
	grip.Info(m)
}
```

Again, it's a nit and more a stylistic question than anything else. Looks good otherwise.